connect/0.8/README.md

```diff
@@ -37,6 +37,7 @@ where `connect` is the name of your existing container. The shell will be set up
 # Environment variables
 
 The Debezium Kafka image uses several environment variables when running a Kafka broker using this image.
+Please also see http://kafka.apache.org/documentation/#connect_running[the documentation] of Kafka Connect to learn more about the specific settings.
```

Comment:
It looks like links are of the format `[the documentation](http://kafka.apache.org/documentation/#connect_running)` here?

Reply:
@psycotica0-shopify You are right, thanks!

Reply:
Agh, yes, the constant confusion between Markdown and AsciiDoc. I'll adjust. Thanks for spotting, @psycotica0-shopify!